### PR TITLE
Fix some typos and wording

### PR DESCRIPTION
**Pull request by tehsphinx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -444,9 +444,6 @@ Please take a look at [Lock Implementation](#lock-implementation) section for im
 ### Lock
 Lock sets a lock for the given key. Acquired lock is only for the key in this DMap.
 
-It returns immediately if it acquires the lock for the given key. Otherwise, it waits until deadline.
```
Comment (tehsphinx) [inline README.md]: This line is repeated right after the example below

File: README.md [hunk 2]
```diff
@@ -784,7 +781,7 @@ the following algorithm:
 Equivalent of`SETNX` command in Olric is `PutIf(key, value, IfNotFound)`. Lock and LockWithTimeout commands are properly implements
 the algorithm which is proposed above. 
 
-You should know that this implementation is a subject of the clustering algorithm. Olric is an AP product. So there is no guarantee about reliability. 
+You should know that this implementation is subject to the clustering algorithm. Olric is an AP product. So there is no guarantee about reliability.
```
Comment (tehsphinx) [inline README.md]: don't nail me on this one. I'm not a native english speaker.

File: README.md [hunk 3]
```diff
@@ -681,18 +678,18 @@ Olric has a component called **fsck** which is responsible for keeping underlyin
...
+A delete operation works the same way.
 
 The data(distributed map objects) in the fragmented partition is moved slowly to the primary owner by **fsck** goroutine. Until the move is done,
 the data remains available on the previous owners. DMap methods use this list to query data on the cluster.
 
-*Please note that, multiple partition owner is an undesirable situation and the fsck component is designed to fix that in a short time.*
+*Please note that, 'multiple partition owners' is an undesirable situation and the fsck component is designed to fix that in a short time.*
 
-When you call **Start** method of Olric, it starts background services with a TCP server.
+When you call **Start** method of Olric, it starts a few background services with a TCP server.
```
Comment (tehsphinx) [inline README.md]: maybe `some` instead?

**Comment by tehsphinx** [comment]
Sorry for the bad commit names.

**Comment by buraksezer** [comment]
@tehsphinx Thank you very much for your contribution. Your PRs are welcome. Please don't hesitate to ask questions about the design and possible integration problems.

**Pull request merged**